### PR TITLE
Help Center: Publish in all locales

### PR DESCRIPTION
**Pull request by escapemanuele**
#### Proposed Changes

This PR publishes the Help Center to all locales.

#### Testing Instructions

* Checkout branch
* 'yarn dev --sync` from `apps/editing-toolkit`
* Set wordpress in another locale
* Check that translations work

### Screenshot for translation

image

### CODE REVIEW

File: packages/help-center/src/components/help-center-sibyl-articles.tsx [hunk 1]
```diff
@@ -81,7 +83,15 @@ export function SibylArticles( { message = '', supportSite }: Props ) {
...
 					<li key={ article.link }>
-						<Link to={ getPostUrl( article as Article, message ) }>
+						<Link
+							onClick={ ( event ) => {
+								if ( 'en' !== locale ) {
+									event.preventDefault();
+									window.open( article.link, '_blank' );
+								}
+							} }
+							to={ getPostUrl( article as Article, message ) }
```
Comment (alshakero) [inline packages/help-center/src/components/help-center-sibyl-articles.tsx]: WDYT about moving this logic into `getPostURL` then adding `target={'en' !== locale ? 'blank' : ''`. It's more accessible this way, people will know where they're going before they click.

**Review by alshakero — APPROVED** [review]
Tiny comment.

File: packages/help-center/src/components/help-center-sibyl-articles.tsx (hunk 1 above)
Comment (escapemanuele) [reply packages/help-center/src/components/help-center-sibyl-articles.tsx]: I've addressed this in the latest commit

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7441749

Thank you @escapemanuele for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7441749).

**Pull request merged**